.github/workflows/ci.yml

```diff
@@ -27,9 +26,10 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
+          pip install -r ${{ matrix.path }}/requirements.txt --no-cache-dir
           pip uninstall -y jina
           pip install jina[cicd,test,http,hub] requests --no-cache-dir
-          pip install -r ${{ matrix.path }}/requirements.txt --no-cache-dir
+          pip install -r pokedex-with-bit/requirements.txt --no-cache-dir
```

Comment:
no, can u add an if matrix_path == pokedex_with_bit install `tensorflow 2.1` or something like this

Reply:
After discussing with the team we thought that add a CI exception for one test only might be sketchy, so I installed the specific version directly on the test, what you think?

Reply:
It is okey